distributed/comm/tcp.py

```diff
@@ -43,7 +43,14 @@
 logger = logging.getLogger(__name__)
 
 
-C_INT_MAX = 256 ** ctypes.sizeof(ctypes.c_int) // 2 - 1
+# Workaround for OpenSSL 1.0.2.
+# Can drop with OpenSSL 1.1.1 used by Python 3.10+.
+# ref: https://bugs.python.org/issue42853
+if sys.version_info < (3, 10):
+    OPENSSL_MAX_CHUNKSIZE = 256 ** ctypes.sizeof(ctypes.c_int) // 2 - 1
+else:
+    OPENSSL_MAX_CHUNKSIZE = 256 ** ctypes.sizeof(ctypes.c_size_t) - 1
```

Comment:
Is the comment about dropping this still true or why do we need another constant? Or is the py3.10 constant simply max integer and therefore a noop or smth like that?

Reply:
Yeah exactly in the Python 3.10 case `OPENSSL_MAX_CHUNKSIZE` is the maximum value `size_t` can contain. IOW the largest integer. So is effectively a no-op.